docker/bookkeeper/Dockerfile

```diff
@@ -8,7 +8,7 @@
 #     http://www.apache.org/licenses/LICENSE-2.0
 #
 
-FROM java:8
+FROM openjdk:8-alpine
```

Comment:
Could you try `openjdk:8-jre-alpine` which what ZK uses?  I just noticed that fact.   Assumedly the difference is that the the JRE image doesn't have JDK stuff like `javac`.